soh/soh/Enhancements/randomizer/ShuffleFairies.cpp

```diff
@@ -179,4 +180,228 @@ void ShuffleFairies_UnregisterHooks() {
+    locationTable[RC_SHADOW_TEMPLE_MQ_BEAMOS_STORM_FAIRY] =             Location::Fairy(RC_SHADOW_TEMPLE_MQ_BEAMOS_STORM_FAIRY,             RCQUEST_MQ,     RCAREA_SHADOW_TEMPLE,         SCENE_SHADOW_TEMPLE,                TWO_ACTOR_PARAMS(0x1000, 55),    "MQ Beamos Song of Storms Fairy",                     "MQ Beamos Song of Storms Fairy",                     RHT_SHADOW_TEMPLE_BEAMOS_STORM_FAIRY,                SpoilerCollectionCheck::RandomizerInf(RAND_INF_SHADOW_TEMPLE_MQ_BEAMOS_STORM_FAIRY));
+    locationTable[RC_SHADOW_TEMPLE_MQ_PIT_STORM_FAIRY] =                Location::Fairy(RC_SHADOW_TEMPLE_MQ_PIT_STORM_FAIRY,                RCQUEST_MQ,     RCAREA_SHADOW_TEMPLE,         SCENE_SHADOW_TEMPLE,                TWO_ACTOR_PARAMS(0x1000, 3342),  "MQ Pit Room Song of Storms Fairy",                   "MQ Pit Room Song of Storms Fairy",                   RHT_SHADOW_TEMPLE_PIT_STORM_FAIRY,                   SpoilerCollectionCheck::RandomizerInf(RAND_INF_SHADOW_TEMPLE_MQ_PIT_STORM_FAIRY));
+    locationTable[RC_SHADOW_TEMPLE_MQ_WIND_HINT_SUN_FAIRY] =            Location::Fairy(RC_SHADOW_TEMPLE_MQ_WIND_HINT_SUN_FAIRY,            RCQUEST_MQ,     RCAREA_SHADOW_TEMPLE,         SCENE_SHADOW_TEMPLE,                TWO_ACTOR_PARAMS(0x1000, -127),  "MQ Wind Hint Sun's Song Fairy",                      "MQ Wind Hint Sun's Song Fairy",                      RHT_SHADOW_TEMPLE_WIND_HINT_SUN_FAIRY,               SpoilerCollectionCheck::RandomizerInf(RAND_INF_SHADOW_TEMPLE_MQ_WIND_HINT_SUN_FAIRY));
+    locationTable[RC_BOTTOM_OF_THE_WELL_MQ_CELL_SUN_FAIRY] =            Location::Fairy(RC_BOTTOM_OF_THE_WELL_MQ_CELL_SUN_FAIRY,            RCQUEST_MQ,     RCAREA_BOTTOM_OF_THE_WELL,    SCENE_BOTTOM_OF_THE_WELL,           TWO_ACTOR_PARAMS(0x1000, -437),  "MQ East Cell Sun's Song Fairy",                      "MQ East Cell Sun's Song Fairy",                      RHT_BOTTOM_OF_THE_WELL_MQ_CELL_SUN_FAIRY,            SpoilerCollectionCheck::RandomizerInf(RAND_INF_BOTTOM_OF_THE_WELL_MQ_CELL_SUN_FAIRY));
+    locationTable[RC_BOTTOM_OF_THE_WELL_MQ_BASEMENT_SUN_FAIRY] =        Location::Fairy(RC_BOTTOM_OF_THE_WELL_MQ_BASEMENT_SUN_FAIRY,        RCQUEST_MQ,     RCAREA_BOTTOM_OF_THE_WELL,    SCENE_BOTTOM_OF_THE_WELL,           TWO_ACTOR_PARAMS(0x1000, -1458), "MQ Basement Sun's Song Fairy",                       "MQ Basement Sun's Song Fairy",                       RHT_BOTTOM_OF_THE_WELL_MQ_BASEMENT_SUN_FAIRY,        SpoilerCollectionCheck::RandomizerInf(RAND_INF_BOTTOM_OF_THE_WELL_MQ_BASEMENT_SUN_FAIRY));
+}
+
+static RegisterShipInitFunc initFunc(Rando::StaticData::RegisterFairyLocations, { "ON_BOOT" });
```

Comment:
putting `"ON_BOOT"` in there doesn't actually do anything, is the idea to start a pattern where we put `"ON_BOOT"` in there to note only running on boot is intentional? if not then just changing this to

```suggestion
static RegisterShipInitFunc initFunc(Rando::StaticData::RegisterFairyLocations);
```

would be good

Reply:
Yeah I guess the ON_BOOT isn't really necessary. If we need to re-run just these later we can always add a different string back in there.

Reply:
Should be fixed now